cmd/dfuseeos/cli/init.go

```diff
@@ -53,19 +53,17 @@ func dfuseInitE(cmd *cobra.Command, args []string) (err error) {
 
 	if runProducer {
 		userLog.Printf("")
-		userLog.Printf("Here the key pair controlling 'eosio' to interact with your local chain:")
+		userLog.Printf("Here are the key pair controlling 'eosio' to interact with your local chain:")
 		userLog.Printf("")
 		userLog.Printf("  Public Key:  EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV")
 		userLog.Printf("  Private Key: 5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3")
 	} else {
 		userLog.Printf("")
-		userLog.Printf("IMPORANT: Move the remote network's 'genesis.json' file in './mindreader' directory")
+		userLog.Printf("IMPORANT: Move the remote network's 'genesis.json' file in the './mindreader' directory")
 	}
 
 	userLog.Printf("")
-	userLog.Printf("Initialization completed, to kickstart your environment run:")
-	userLog.Printf("")
-	userLog.Printf("  dfuseeos start")
+	userLog.Printf("Initialization completed! To kickstart your environment, run: 'dfuseeos start'")
```

Comment:
The old version was clearer in my opinion because `dfuseeos start` was more "in your face", now it's kinda meld into the `Initialization complete` and makes it less clear that you need to run the command.

What was the reasoning for the change?

Reply:
+1 for Matt

Reply:
True, I was trying to mimic the "purge" line that says: `Purged data. Start a fresh instance with 'dfuseeos start'`. On a scale of 1 to 10, I'm a 3-4 on this change so I don't mind if you move it back to what it was. I'm just trying to keep things "standard" in the way we display console comments to the user.